t/features/builtins/methods.t

```diff
@@ -309,5 +309,47 @@ use _007::Test;
     is-result $ast, "True\nFalse\n", "String.contains() works";
 }
 
+{
+    my $program = q:to/./;
```

Comment:
Hm, I like the tests themselves, and they were more than I expected what with `X::Regex::InvalidMatchType` and all &mdash; but I'm worried that we don't have any tests for the parsing aspects of regexes.

Don't worry; I tested them manually, and it works fine even when fed through the parser:

    $ FLAG_007_REGEX=1 bin/007 -e='say(/"foo"/.fullmatch("foo"))'
    True
    $ FLAG_007_REGEX=1 bin/007 -e='say(/"foo"/.fullmatch("fool"))'
    False

But it feels a bit odd that we're introducing new syntax behind a feature flag without any test coverage at all.